utils/cosmos.ts

```diff
@@ -236,8 +288,6 @@ export async function signCreateRoyltyConfig (
       })
     }
   } catch (err) {
-    // Don't throw on royalty create, not critical for now
-    // eslint-disable-next-line no-console
```

Comment:
Should keep this rule

Reply:
[♻️ Refactor ISCN library imports to consolidate and simplify usage](https://github.com/likecoin/nft-book-press/pull/320/commits/ad11d3b863a48cea1c1ee76b3766ab507ad1a4e2)